funnel/templates/project.html.jinja2

```diff
@@ -83,8 +83,18 @@
             <div id="boxoffice-widget"><p class="mui--text-body1">Loading...</p></div>
           {% elif project.allow_rsvp -%}
             <h2 class="mui--text-display1 mui--text-left project-section__headline">{% trans %}Attending this event?{% endtrans %}</h2>
+            {% if current_rsvp is not none %}
+            <p>Your current status: {{ current_rsvp.state.label.title }}
```

Comment:
`{% trans %}` needed. You'll have to incorporate the title as a variable in the trans tag so the whole paragraph is sent for translation.

Reply:
Missing `</p>`